force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,27 @@
+trigger AccountTrigger on Account (before insert, after insert) {
```

Comment:
Try to separate each method into it's own if statement instead of having them all into one if-else